include/odp/api/spec/ipsec.h

```diff
@@ -335,6 +335,28 @@ typedef struct odp_ipsec_capability_t {
 	 */
 	uint32_t max_cls_cos;
 
+	/**
+	 * Scheduled destination queue support
+	 *
+	 * This defines whether scheduled queues are supported as destination
+	 * queue.
```

Comment:
This API change looks ok to me. I would just remove the "this defines..." sentences since they do not seem to add anything to what is otherwise anyway said.

Reply:
Ack. Will remove these sentences in next version.